src/styles/pages/roar-privacy-tos.scss

```diff
@@ -1,8 +1,23 @@
 .layout.privacy-tos-page {
+  header {
+    position: initial;
+  }
+
```

Comment:
Let's make the column centered within `main` like so

```scss
main {
  display: flex;
  flex-direction: column;
  align-items: center;
}
```

Reply:
add styles for main as suggested